packages/core-bridge/src/errors.rs

```diff
@@ -65,6 +67,13 @@ impl CustomError for OnceCell<Root<JsFunction>> {
 /// It expects a single argument, an object with the various Error constructors.
 /// This is a very common pattern in Neon modules.
 pub fn register_errors(mut cx: FunctionContext) -> JsResult<JsUndefined> {
+    // Don't do anything if errors are already registered
```

Comment:
Your solution is prone to races if this function is called concurrently.

I'd just use [`set`](https://docs.rs/once_cell/latest/once_cell/sync/struct.OnceCell.html#method.set) here and early return on error.

Reply:
Sounds good, will update